src/helpers/buildWalletSections.tsx

```diff
@@ -105,6 +111,51 @@ const withPositionsSection = (isLoadingUserAssets: boolean) => {
   return [];
 };
 
+const withClaimablesSection = (isLoadingUserAssets: boolean) => {
+  // check if the feature is enabled
+  const claimablesEnabled = getExperimetalFlag(CLAIMABLES) || getRemoteConfig().claimables;
+  if (!claimablesEnabled) return [];
+
+  const { accountAddress: address, nativeCurrency: currency } = store.getState().settings;
+  const claimables: Claimable[] | undefined = queryClient.getQueryData(
+    claimablesQueryKey({ address, currency, testnetMode: getIsHardhatConnected() })
+  );
+
+  const result: ClaimableExtraData[] = [];
+  let totalNativeValue = '0';
+  claimables?.forEach(claimable => {
+    totalNativeValue = add(totalNativeValue, claimable.value.nativeAsset.amount);
```

Comment:
is the nativeAsset.amount ever undefined or could it cause issues here? I think we just want to safeguard these additions as much as possible.

Reply:
it should never cause issues but i added a safeguard anyway